src/project.mk

```diff
@@ -78,7 +78,7 @@ ifneq ($(REALM_ANDROID),)
   PROJECT_CFLAGS += -fPIC -DPIC
   # android.toolchain.cmake has `-fsigned-char` by default, we have to use the same
   # to avoid lto linking problems.
-  CFLAGS_OPTIM = -Os -flto -ffunction-sections -fdata-sections -DNDEBUG -fsigned-char -fvisibility=hidden
+  CFLAGS_OPTIM = -Os -DNDEBUG -fsigned-char -fvisibility=hidden
```

Comment:
`-ffunction-sections` and `-fdata-sections` are enabled by default for our android cmake https://github.com/realm/realm-java/blob/master/realm/realm-library/src/main/cpp/android.toolchain.cmake#L1356 . Will it have any impact on the size of final linking result?